supercursor-framework/src/integrations/output-formatter.ts

```
@@ -0,0 +1,360 @@
+          : `Error: ${errorMessage}`;
+
+        if (context && Object.keys(context).length > 0) {
+          formatted += `\nContext: ${JSON.stringify(context, null, this.config.indentSize)}`;
+        }
+
+        if (errorStack && process.env.NODE_ENV === 'development') {
+          formatted += `\n${errorStack}`;
+        }
+
+        return formatted;
+    }
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)